app.js

```diff
@@ -1,24 +1,207 @@
+    tags: ['XML', 'CSS', 'Ruby'],
+    demoLink: '',
+    codeLink: '',
+  },
+  {
+    name: 'Multi Post Stories',
+    img: 'images/ex.png',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    description2:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industrys standard.',
+    tags: ['XML', 'CSS', 'Ruby'],
+    demoLink: '',
+    codeLink: '',
+  },
+  {
+    name: 'Multi Post Stories',
+    img: 'images/ex.png',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    description2:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industrys standard.',
+    tags: ['XML', 'CSS', 'Ruby'],
+    demoLink: '',
+    codeLink: '',
+  },
+];
+
+//  Generate Work section from Js file
+
+const works = document.getElementById('card-holder');
+
+for (let i = 0; i < projects.length; i += 1) {
+  const cardHolder = document.createElement('div');
+  cardHolder.innerHTML = `
+  <div class="info-holder">
+    <h2 class="project-heading white-font">
+    ${projects[i].name}
+    </h2>
+    <p class="card-description white-font">
+    ${projects[i].description2}
+    </p>
+    <ul class="tag-list white-font">
+
+      <li class='card-feature-items'>${projects[i].tags[0]}</li>
+      <li class='card-feature-items'>${projects[i].tags[1]}</li>
+      <li class='card-feature-items'>${projects[i].tags[2]}</li>
+    </ul>
+  </div>
+  <div class='button-section'>
+    <a
+      id='cardButton${i}'
+      class='pop card-button dark-font seeInfo'
+      href='#popUp'
+      >See Project</a
+    >
+  </div>
+    `;
+  cardHolder.classList.add('work-card');
+  works.appendChild(cardHolder);
+}
+
+//  Deatils Popup window
+
+const popup = document.createElement('div');
+function createPopup(nbr) {
+  popup.innerHTML = `
+  <div id="popup-container" class="popup-window">
+  <div class="popup-holder">
+    <div class="popup-close-btn-holder">
+      <div class="title-xbtn">
+        <h1 id="popup-header" class="project-heading project-heading-pop">${projects[nbr].name}</h1>
+        <svg
+          id="x-popup"
+          class=".xBtn"
+          xmlns="http://www.w3.org/2000/svg"
+          width="32"
+          height="32"
+          viewBox="0 0 24 24"
```

Comment:
- [ ] Going by the Figma design, please ensure to give some space on the left and right of the popup so that the blurred background image can be more visible just like the Figma design
## Figma design
![image](https://user-images.githubusercontent.com/46839548/231908213-3f56afde-e0b1-47e0-adb5-cba39e00c764.png)
## Your Design
![image](https://user-images.githubusercontent.com/46839548/231908258-726b7385-0604-486e-a122-13946ade36e5.png)